website/content/api-docs/auth/jwt.mdx

```diff
@@ -134,11 +134,11 @@ entities attempting to login. At least one of the bound values must be set.
 - `bound_subject` `(string: <optional>)` - If set, requires that the `sub`
   claim matches this value.
 - `bound_claims` `(map: <optional>)` - If set, a map of claims (keys) to match against respective claim values (values).
-  The expected value may be a single string or a list of strings. The interpretation of the bound
+  Each expected value may be a string, integer, boolean or a list of strings. The interpretation of the bound
   claim values is configured with `bound_claims_type`. Keys support [JSON pointer](/vault/docs/auth/jwt#claim-specifications-and-json-pointer)
   syntax for referencing claims.
 - `bound_claims_type` `(string: "string")` - Configures the interpretation of the bound_claims values.
-  If `"string"` (the default), the values will treated as string literals and must match exactly.
+  If `"string"` (the default), the values will be treated as literals and must match exactly.
```

Comment:
Here `bound_claims_type` created 2 kinds: `string` and `glob`. However, in reality, they are `glob` and `non-glob`. If the value is an integer or a boolean, the default interpretation (string) will work just fine. That is why I converted `string literals` to just `literals`.

Reply:
This makes sense to me -- thanks for updating this!

Reply:
+, thanks!